### PR TITLE
54 show story after submitting the form

### LINKED ISSUE #25: As a user, when seeing my story, I will have a button to take me back to the beginning.
Fulfilled by Task #14

### LINKED ISSUE #30: Landing Page
- [x] #23
- [x] #32
- [x] #29
- [x] #24
- [x] #25
- [x] #26
- [x] #27
- [x] #28

### LINKED ISSUE #54: Show story after submitting the form
- [ ] First page should display the form with 'Submit' and 'Reset' buttons. 
- [ ] After clicking on 'Submit', story should be displayed in place of form.
- [ ] Should be able to go back to 'Form' when clicked on button 'Play Again'.

### PR DESCRIPTION
**Pull request by ss-deep**
## Description

 -Initially page displays the form with 'Submit' and 'Reset' buttons.
 -After clicking on 'Submit', story is displayed in place of form.
 -Goes back to 'Form' when clicked on button 'Play Again'.

(It needs styling to look neat and nice.)

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|     | :sparkles: New feature     |
| ✓ | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,11 +1,10 @@
-
 body{
     width: 100vw;
     margin: 0; 
     box-sizing: border-box;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
```
Comment (terrifricker) [inline style.css:5]: great change!

File: style.css [hunk 2]
```diff
@@ -27,41 +26,40 @@ body{
     font-size: 1.2rem;
```
Comment (terrifricker) [inline style.css:24]: Could take font-family out of \<story> because it is now in \<body>.

Comment (ss-deep) [reply style.css:24]: oh, missed it. i will change.

File: style.css (hunk 1 above)
Comment (ss-deep) [reply style.css:5]: Thanks

File: second.html [hunk 3]
```diff
@@ -92,12 +92,12 @@ <h1 class="game-title">
                         <input type="text" id="password" name="password" placeholder="">
                     </div>
                     <div class="buttons">
-                        <button type="submit">Reset</button>
+                        <button type="reset">Reset</button>
```
Comment (terrifricker) [inline second.html:95]: Good catch!

File: app.js [hunk 4]
```diff
@@ -5,6 +5,7 @@ const story = document.querySelector(".story");
 // have the form listen for the submit event
 wordForm.addEventListener("submit", (e) => {
     e.preventDefault();
+    wordForm.classList.add('story');
```
Comment (terrifricker) [inline app.js]:
```suggestion
    wordForm.classList.add('hidden');
```

File: app.js [hunk 5]
```diff
@@ -37,4 +38,16 @@ wordForm.addEventListener("submit", (e) => {
   
     // place the story into the landing page
     story.innerHTML = storyHTML;
-});
\ No newline at end of file
+    story.classList.remove('story');
```
Comment (terrifricker) [inline app.js]:
```suggestion
    story.classList.remove('hidden');
```

File: app.js [hunk 6]
```diff
@@ -37,4 +38,16 @@ wordForm.addEventListener("submit", (e) => {
...
+
+    //creates button to go back to the form
+    let resetButton = `<br><button id="game-reset" onclick="resetGame()">Play Again</button>`;
+    story.innerHTML += resetButton;
+});
+
+function resetGame() {
+    story.classList.add('story');
```
Comment (terrifricker) [inline app.js]:
```suggestion
    story.classList.add('hidden');
```

File: app.js [hunk 7]
```diff
@@ -37,4 +38,16 @@ wordForm.addEventListener("submit", (e) => {
...
+    story.innerHTML = '';
+    wordForm.reset();
+    wordForm.classList.remove('story');
```
Comment (terrifricker) [inline app.js]:
```suggestion
    wordForm.classList.remove('hidden');
```

File: second.html [hunk 8]
```diff
@@ -34,7 +34,7 @@ <h1 class="game-title">
     <body>
         <div class="container">
             <section class="word-form">
-                <form action="#">
+                <form class="input-form" name="input-form" action="#">
```
Comment (terrifricker) [inline second.html:37]: Using a name attribute on a form was new to me.  I learned something.

File: second.html [hunk 9]
```diff
@@ -92,12 +92,12 @@ <h1 class="game-title">
                         <input type="text" id="password" name="password" placeholder="">
                     </div>
                     <div class="buttons">
-                        <button type="submit">Reset</button>
+                        <button type="reset">Reset</button>
                         <button type="submit">Submit</button>
                     </div>
                 </form>
+                <section class="story"></section>
```
Comment (terrifricker) [inline second.html]:
```suggestion
                <section class="story hidden"></section>
```

File: style.css
Comment (terrifricker) [inline style.css:1]: Add
```
.hidden {
    display: none;
}
```
to end of style.css file.

File: style.css [hunk 10]
```diff
@@ -19,49 +18,45 @@ body{
...
     margin-left: 39%;
-    background-color: rgb(80, 231, 168);
     background-size: 5px;
 }
 .story {
-    font-family: Verdana, Geneva, Tahoma, sans-serif;
     font-size: 1.2rem;
     line-height: 2.2rem;
     background-color: #D9D9D9;
+    display: none;
```
Comment (terrifricker) [inline style.css]:
```suggestion
```

Comment (terrifricker) [reply style.css]: Remove display: none; from story class and instead create a new class named 'hidden' to the bottom of this file.

**Review by terrifricker — CHANGES_REQUESTED** [review]
It's looking so good, @ss-deep 

I've sent you files with comments about your javascript through discord.  Your reset button is perfect!  Your logic for using display: none; is perfect, too.  I just suggest that instead of adding display: none; to your story class, you create a new class called hidden.

```
hidden {
    display: none;
}
```
adding and removing 'hidden' from the classLists of the form and story instead.

For the first time, I figured out how to make the suggestions right in the code, so you can just click a button to accept the changes one by one, or even as a batch all together.  The instructions are found at: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request

**Review by terrifricker — APPROVED** [review]
Approving as is!  Any changes decided on can be done as refactoring.

**Pull request merged**